data/schema/units/abilities.cfg

```diff
@@ -133,10 +144,23 @@
 {BASED_ON_SPECIAL "poison"}
 {BASED_ON_SPECIAL "slow"}
 {BASED_ON_SPECIAL "petrifies"}
+{IMMUNE_TO_SPECIAL "unattacks"}
+{IMMUNE_TO_SPECIAL "unswarm"}
```

Comment:
It makes sense for consistency with the other ones, but "immune to swarm" is a bit weird.

Having severed all but one head of the great hydra, the foolish hero threw his shield to the ground and beckoned the stunned beast on. "Come weakling, do not let the legends claim that I merely slew an injured beast. Have five goes at biting me, as if you still had all of your heads."